hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/subscription/validator/SubscriptionValidator.java

```diff
@@ -23,27 +23,27 @@
     private final TopicService topicService;
     private final SubscriptionRepository subscriptionRepository;
     private final Optional<EndpointOwnershipValidator> endpointOwnershipValidator;
+    private final Optional<SubscriptionEndpointValidator> subscriptionEndpointValidator;
```

Comment:
We already have an endpoint validator: `pl.allegro.tech.hermes.api.constraints.EndpointAddressValidator`. Can we extend it instead of introducing another one? I think it will be clearer to have one place where this validation is done (especially considering the fact that in most cases the validator has to be aware of the endpoint protocol - `EndpointAddressValidator` already understands that we support different protocols).